core/bin/external_node/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zksync_external_node"
-version = "0.1.0"
+version = "22.1.0" # x-release-please-version
```

Comment:
Checked this annotation by running `release-please` locally [as per the troubleshooting guide](https://github.com/googleapis/release-please/blob/main/docs/troubleshooting.md); seems to work.

Reply:
We could configure release please to update all crates, but ill leave this decision to @popzxc

Reply:
Would the versions of library crates necessarily correspond to EN / server versions? In general, release-please supports Cargo workspaces, but it looks somewhat convoluted judging by docs (and there may be other reasons why it's not used already 🙃).